legion/engine/core/engine/engine.hpp

```diff
@@ -3,7 +3,10 @@
 #include <core/engine/module.hpp>
 #include <core/types/primitives.hpp>
 #include <core/types/meta.hpp>
+#include <core/async/spinlock.hpp>
+#include <core/containers/pointer.hpp>
 #include <core/logging/logging.hpp>
+
 #include <argh.h>
```

Comment:
is there a reason why a space was added here?

Reply:
yeah, to separate the legion includes from external includes